eyecite/regexes.py

```diff
@@ -106,8 +106,10 @@ def short_cite_re(regex):
 
 MONTH_REGEX = r"""
     (?P<month>
-        Jan\.|Feb\.|Mar\.|Apr\.|May|June|
-        July|Aug\.|Sept\.|Oct\.|Nov\.|Dec\.
+        Jan\.|January|Feb\.|February|Mar\.|March|
+        Apr\.|April|May|June|July|Aug\.|August|
+        Sept\.|September|Oct\.|October|
+        Nov\.|November|Dec\.|December
     )
```

Comment:
I think we can improve this regex slightly.  I see you didnt provide abbreviations for June and July.

```
MONTH_REGEX = r"""
     (?P<month>
      Jan(?:uary)?\.?   |
      Feb(?:ruary)?\.?  |
      Mar(?:ch)?\.?     |
      Apr(?:il)?\.?     |
      May\.?            |
      Jun(?:e)?\.?      |
      Jul(?:y)?\.?      |
      Aug(?:ust)?\.?    |
      Sep(?:t(?:ember)?)?\.? |
      Oct(?:ober)?\.?   |
      Nov(?:ember)?\.?  |
      Dec(?:ember)?\.?
     )
"""
```
I found a number of examples of Jun. being abbreviated. 

`…Corp., 2009 U.S. Dist. LEXIS 52214, at *4 (E.D. Pa. Jun. 18, 2009). Put another way, discovery may not …`
`L 179797 (E.D. Pa. Mar. 25, 1999). Ther`
`WL 363921, at *2 (E.D.Pa. Jul. 1,1996). Because a § 2255`

Reply:
Excellent, thanks